src/riscv/lib/src/jit/builder/outcome_map.rs

```diff
@@ -26,6 +26,18 @@ use std::ops::Range;
 
 use crate::jit::builder::instr_map::InstrId;
 
+/// Priority of taking an instruction outcome based on its likelihood.
+/// As detailed in RISC-V Control Transfer Instructions specification (2.5),
```

Comment:
```suggestion
///
/// As detailed in RISC-V Control Transfer Instructions specification (2.5),
```
(mega nit - makes the documentation easier to read)

Reply:
done [here](https://github.com/tezos/riscv-pvm/compare/91b9cd4e65b5841ae2c1788382731cd1d0b79534..0e83f1e1eae7489751565660e0f13c4f8a518e6a)